mix.exs

```diff
@@ -49,10 +49,18 @@ defmodule Tableau.MixProject do
 
       # dev
       {:ex_doc, ">= 0.0.0", only: :dev},
-      {:styler, "~> 0.9", only: :dev}
+      {:styler, styler_constraint(), only: :dev}
     ]
   end
 
+  defp styler_constraint do
```

Comment:
CI runs format only with a single version so I think it's okay to keep this static. 

That workflow uses 1.15 tho, so that should get updated to 1.17

Reply:
Yes and no. Anyone who develops on tableau that uses the minimum supported (1.12) will likely have an install failure because of this (I *think* it would also make the release action, using 1.14, fail).

Making it static and bumping the minimum supported Elixir version to 1.15 would definitely simplify things across the board.

Please advise.

Reply:
Minimum version can be bumped, I can almost guarantee no one building a site with Tableau would be using Elixir 1.12. That just happened to be the current version of Elixir when the project was created.

Reply:
OK. Because of that, I’m going to roll this and #95 together so that the builds don't fail.